dubbo-cluster/src/test/java/com/alibaba/dubbo/rpc/cluster/loadbalance/LoadBalanceTest.java

```diff
@@ -71,6 +73,14 @@ public void setUp() throws Exception {
         invoker3 = EasyMock.createMock(Invoker.class);
         invoker4 = EasyMock.createMock(Invoker.class);
         invoker5 = EasyMock.createMock(Invoker.class);
+        invocationList = new ArrayList<Invocation>();
+        for(int i=0; i<1000; i++){
+            Invocation  invocation1 = EasyMock.createMock(Invocation.class);
+            EasyMock.expect(invocation1.getMethodName()).andReturn("method1").anyTimes();
+            EasyMock.expect(invocation1.getArguments()).andReturn(new Object[]{randomString(3),randomString(5)}).anyTimes();
+            EasyMock.replay(invocation1);
```

Comment:
Here, we have changed test framework to Mockito, so could you change it to mockito again? Thanks.